proof_of_stake/src/validation.rs

```diff
@@ -1469,6 +1494,62 @@ where
         }
     }
 
+    fn validator_commission_rate(
+        constants: &Constants,
+        errors: &mut Vec<Error<Address, TokenChange, PublicKey>>,
+        new_validators: &mut HashMap<Address, NewValidator<PublicKey>>,
+        address: Address,
+        data: Data<CommissionRates>,
+    ) {
+        match (data.pre, data.post) {
+            // Should this case ever happen since commission rate should be init
+            // at genesis? (same w consensus key tho)
```

Comment:
yeah, only genesis validators will have this set - validators created later via a tx will go through this validation